### PR TITLE
move download settings to own PreferencePane

### LINKED ISSUE #291: Splitting Advanced Tab to 2 different tabs
As of now, Advanced setting is filled up with many settings and it would need a space where it would be hidden behind the bottom bar.

Suggestion: Split the options into two section.

Ekran Resmi 2022-09-17 23 28 00

### PR DESCRIPTION
**Pull request by leon-wolf**
resolves #291 

- move downloads related settings to own preference pane

### CODE REVIEW

File: Xcodes/Frontend/Preferences/DownloadPreferencePane.swift [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import AppleAPI
+import SwiftUI
+
+struct DownloadPreferencePane: View {
```
Comment (MattKiazyk) [inline Xcodes/Frontend/Preferences/DownloadPreferencePane.swift:4]: Looks like `DownloadPreferencePane` wasn't added to the project.

**Review by MattKiazyk — CHANGES_REQUESTED** [review]
@leon-wolf Thanks for this - looks super clean - just a fix up to make sure you add `DownloadPreferencePane` to the project

File: Xcodes/Frontend/Preferences/DownloadPreferencePane.swift (hunk 1 above)
Comment (leon-wolf) [reply Xcodes/Frontend/Preferences/DownloadPreferencePane.swift:4]: should be added in the latest commit. I don't really know how it was not added :D

**Review by MattKiazyk — APPROVED** [review]

**Comment by MattKiazyk** [comment]
Thanks @leon-wolf for all the work!

**Pull request merged**